sdk/servicebus/service-bus/test/stress/app/src/serviceBusStressTester.ts

```diff
@@ -160,7 +160,9 @@ export class ServiceBusStressTester {
         }
       } catch (error) {
         this.sendInfo.numberOfFailures++;
-        this.trackError("send", error);
+        if (error instanceof Error) {
```

Comment:
don't need `if` here as unknown is allowed now.

Reply:
oops